### PR TITLE
Update circle ci config to latest globality-build

### PR DESCRIPTION
**Pull request by JP-Globality**

### CODE REVIEW

**Comment by JP-Globality** [comment]
Also had to fix the following lint issues as part of the circle ci bump:
microcosm_sqlite/factories.py:6:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_builders.py:8:1: I003 isort expected 1 blank line in imports, found 0
microcosm_sqlite/tests/test_builders.py:16:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_builders.py:17:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_builders.py:18:1: I004 isort found an unexpected blank line in imports
microcosm_sqlite/tests/test_foreign_keys.py:3:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_except.py:3:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_types.py:7:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_factories.py:6:1: I003 isort expected 1 blank line in imports, found 0
microcosm_sqlite/tests/test_factories.py:16:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_factories.py:17:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_stores.py:5:1: I003 isort expected 1 blank line in imports, found 0
microcosm_sqlite/tests/test_stores.py:15:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_stores.py:16:1: I004 isort found an unexpected blank line in imports
microcosm_sqlite/tests/test_dumpers.py:8:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_dataset.py:4:1: I003 isort expected 1 blank line in imports, found 0
microcosm_sqlite/tests/test_dataset.py:11:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_dataset.py:12:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_dataset.py:13:1: I001 isort found an import in the wrong position
microcosm_sqlite/tests/test_dataset.py:14:1: I004 isort found an unexpected blank line in imports
microcosm_sqlite/tests/test_threading.py:8:1: I001 isort found an import in the wrong position

Coming from this commit: https://github.com/globality-corp/microcosm-sqlite/pull/68/commits/6a830cc704ce42e8cba43a5ee742eb1af79bbfc1
lint job failure: https://app.circleci.com/pipelines/github/globality-corp/microcosm-sqlite/45/workflows/746d5fb9-50de-4051-a70b-49217a7477de/jobs/983

**Review by ronanm-globality — APPROVED** [review]

File: entrypoint.sh [hunk 1]
```diff
@@ -20,20 +19,20 @@
...
 
 if [ "$1" = "test" ]; then
-    # Install standard test dependencies; YMMV
-    pip --quiet install \
-        .[test] nose PyHamcrest coverage
-    exec nosetests ${NAME}
+   # Install standard test dependencies; YMMV
+   pip --quiet install \
+       .[test] nose PyHamcrest coverage
+   exec nosetests
```
Comment (ronanm-globality) [inline entrypoint.sh:25]: I guess we'll want to move this to pytest at some point too, but out of scope here!

Comment (JP-Globality) [reply entrypoint.sh:25]: Good thinking - I'll raise a ticket and add it to our backlog

Comment (JP-Globality) [reply entrypoint.sh:25]: Have raised GLOB-54120 for this 👍

**Pull request merged**